test/unit/nedap/utils/test/api.cljc

```diff
@@ -110,3 +113,70 @@
       (gensym)         42
       (gensym)         nil
       [[1 (gensym) 2]] [[1 (gensym) 3]])))
+
+(deftest expect
+  (let [a (atom 0)]
+    (sut/expect (swap! a inc)
+                :to-change @a
+                :from 0
+                :to 1)
+
+    (sut/expect (swap! a inc)
+                (swap! a inc)
+                :to-change @a
+                :from 1
+                :to 3))
+
+  (testing "the macroexpansion evaluates each part exactly once"
```

Comment:
`:to-change` is evaluated exactly twice, which is correct but contradictory w/ the `testing`